src/styles/style.scss

```diff
@@ -0,0 +1,14 @@
+@import 'utils';
```

Comment:
commenting here to not pollute the other `scss` files.

i don't see any hover and cursor styles across the styling.

![image](https://github.com/user-attachments/assets/53364010-88c0-4dcb-9d15-9a7339748641)